spec/request/api/v1/topic/topic_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Topic /api/v1/topics', type: :request do
+  let(:user) { create(:user, confirmed_at: Time.zone.today) }
+  subject do
+    10.times { create(:topic) }
```

Comment:
You can also use `create_list`.

https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#building-or-creating-multiple-records